scripts/gen_c_feature_dictionary.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python3
+
+# Copyright 2024 The Khronos Group Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+from collections import OrderedDict
+
+import argparse
+import sys
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('-features', action='store',
+                        default='',
+                        help='File with OpenCL C features to generate, one per line')
+    parser.add_argument('-o', action='store', default='',
+                        help='Output file in which to store the feature dictionary. stdout is used if no file is provided.')
+
+    args = parser.parse_args()
+
+    features = []
+    if len(args.features) > 0:
+        print('Generating feature dictionaries from: ' + args.features + ' ...\n\n')
```

Comment:
Nitpick: I'd prefer not to have additional newlines here, to be consistent with the other dictionary generation scripts.

```suggestion
        print('Generating feature dictionaries from: ' + args.features)
```